doc/devguide/codebase/code-style.rst

```diff
@@ -6,12 +6,191 @@ Suricata uses a fairly strict coding style. This document describes it.
 Formatting
 ~~~~~~~~~~
 
+clang-format
```

Comment:
The doc diff can be a pain to read.

See https://github.com/roligugus/suricata/blob/clang-format-proposal-v5/doc/devguide/codebase/code-style.rst for the formatted doc.